### ISSUE TITLE
A11y - iframe and image do not have a title when YouTube/Vimeo video is loaded with consent

### ISSUE THREAD

**Issue by timo-haas-gh** [shopware/platform#7003]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.10.1  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

When a YouTube or Vimeo video is embeded via CMS element the title is not shown when "Load video only after confirmation by the user" is used.

*_Expected behaviour:_*

The iframe should have a title and the preview image a title or alt text

*_How to reproduce:_*

Create a video in your shopping experience and select the option "Load video only after confirmation by the user"

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40781**



[created from NEXT-40781, comment 541006]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40781, comment 541223]

**Comment by ravenokko** [comment]
Fixed by [#7636](https://github.com/shopware/shopware/pull/7636)
